contentcuration/contentcuration/utils/files.py

```diff
@@ -23,7 +23,8 @@
 def create_file_from_contents(contents, ext=None, node=None, preset_id=None, uploaded_by=None):
     checksum, _, path = write_raw_content_to_storage(contents, ext=ext)
     with default_storage.open(path, 'rb') as new_file:
-        return File.objects.create(
+        # import ipdb; ipdb.set_trace()
```

Comment:
Delete!

Reply:
This breakpoint is triggered after the mysterious error.